content/community-projects.md

```diff
@@ -14,14 +14,26 @@ use in your own project.
 
 ## Duffle
 <a href="https://duffle.sh" alt="duffle"><img align="right" src="/img/duffle.svg" width="64px" /></a>
-Duffle is a command line tool that allows you to install and manage CNAB bundles.
+Duffle is the reference implementation of the CNAB specification. It is best for
```

Comment:
I do not like this change. You changed the text from something that was accurate and reasonable to something that implies a deficiency in Duffle, and in an area that the original text did not even call out.